art-common/artcommonlib/model.py

```diff
@@ -0,0 +1,159 @@
+from future.utils import as_native_str
```

Comment:
We should remove `doozerlib/model.py` and `elliottlib/model.py`

Reply:
We are in the future. Let's not import it.